### DISCUSSION TITLE
Bug in the wallets

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#6363] [Support]
Hello there!
There appears to be a bug in my wallets section. There are incoming transactions and in the farm section, it says that there is 2 chia mined, however, the total, spendable and pending balances are empty

**Comment by XIII39** [comment]
Is your wallet synced upto and beyond the block that contains the transaction? Not just the Full Node.

I have found that, from time to time, the wallet 'misplaces' a block. If you go to the Full Node tab and do a search for the block in question, if you get an error such as "block not found" doing a View --> Force Reload makes everything show up where it's supposed to be.

**Reply by ghost** [reply]
It doesn't seem to say that. But I did try to force reload it. Now it seems to be stuck at the menu "connecting to the wallet"

**Reply by XIII39** [reply]
Close the Chia GUI, make sure all the processes are ended in the task manager and re-open the GUI

**Reply by ghost** [reply]
Ok, I did that. I even rebooted my computer. Still says 0 XCH, when there are like at least 2 in the transactions

**Reply by ghost** [reply]
Do I need to redo the force reload?

**Reply by gotblindside** [reply]
Same error. performed the same procedure.

![image](https://user-images.githubusercontent.com/83858356/120890355-c1fb4100-c634-11eb-8e3c-74ea2387e8b6.png)

**Comment by BrandtH22** [comment]
@trofoshka , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
